botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
+
+    @staticmethod
```

Comment:
Sure I can change that.

While I have no problem with removing the staticmethod part, I will catch up with you guys to understand the criteria so that I will make confident choice in future. At this moment I see some other staticmethod internal methods in our code base. Not sure when they would be appropriate.

By the way, just share some anecdote in this context for your weekend fun. In my very first unpublished prototype, this function was originally a completely separated (and therefore static) function named ``cloudfront_b64encode()``, because it is neither a ``standard_b64encode()`` nor the another Python builtin ``urlsafe_b64encode()``. Later it was grouped into this CloudFrontSigner class, so I removed that seemingly excessive prefix.

So it will be a normal internal method as ``_cloudfront_b64encode(self, content)`` in next PR.